data/xml/1952.preconf.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="1952.preconf">
+  <volume id="1" ingest-date="2021-09-29">
```

Comment:
I think "preconf" is not a good name. It worked in the context of the MT Archive, but we're in a more global namespace now. I wonder if "earlymt" would make sense for this and others?

@danielgildea and @davidweichiang, do either of you have any input on how best to name these early MT conferences?